Main.java

```diff
@@ -9,7 +11,33 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    Scanner input = new Scanner(System.in);
+		// ask user for the quanity the need for each item
+		System.out.println("How many chromebook chargers?");
+		int chargers = input.nextInt();
+		System.out.println("How many motherboards?");
+		int motherboards = input.nextInt();
+		System.out.println("How many mice?");
+		int mice = input.nextInt();
+		/**
+		double CHROMEBOOK_CHARGER = 34.99;
+		double REPLACEMENT_MOTHERBOARD = 127.50;
+		double COMPUTER_MOUSE = 18.00;
+		*/
```

Comment:
I'm not sure why you didn't use these constants and instead commented them out. This is the PERFECT reason why you would use a constant.